gcc/rust/expand/rust-macro-expand.cc

```diff
@@ -2036,6 +2036,10 @@ class AttrVisitor : public AST::ASTVisitor
 	    return;
 	  }
       }
+    else
+      {
+	std::string mod_file = module.get_filename ();
```

Comment:
what does this line do?

Reply:
Ah, this is just the first step of items loading. I added it so that the function wouldn't be unused but I can remove it for now. Basically, the flow is going to be like this:

```c++
std::string mod_file = module.get_filename ();

if (mod_file.is_empty ()) return;

RAIIFile file_wrap(mod_file);
Linemap *linemap = /* magic */;

Lexer lex(mod_file.c_str (), std::move (file_wrap), linemap);
Parser<Lexer> parser (std::move (lex));

auto items = parser.parse_items ();

this.set_items (items);
```